core/src/main/groovy/noe/server/ApacheDS.groovy

```diff
@@ -52,6 +54,24 @@ abstract class ApacheDS extends ServerAbstract {
 
     portsAvailable()
     CmdCommand cmdCommand = new CmdBuilder<>(start).setWorkDir(new File(apacheDSBin)).build()
+    Map cmdProps = cmdCommand.getEnvProperties()
+    if(platform.isFips()) {
+      //ApacheDS won't start with FIPS
+      def opts = cmdProps.getOrDefault("JAVA_OPTS","")
+      if(opts.size()>0) {
+        opts+=' '
+      }
+      opts+='-Dcom.redhat.fips=false'
```

Comment:
@sgala does this change to JAVA_OPTS get cleaned up / removed when the ApacheDS test has completed.

Reply:
The change disappears when the variable cmdCommand goes out of scope at the end of the method execution.